Makefile

```diff
@@ -67,14 +68,21 @@ test-federation-identity:
 test-storage:
 	go test -count=1 -v storageClient/storageClient_test.go
 
+# Run all Account-V1 Integration tests
+test-account:
```

Comment:
I would flip this, we are more often than not running external tests 
so make test-account the external and then test-account-internal, following the identity pattern

Reply:
i was thinking that if we're running internal tests we want to run all the tests vs only wanting external. what do you think?

Reply:
hmmm we need a pattern then, 
either changing identity to follow the same or renaming 
"test-only-external-account"

Reply:
I agree with Katie but am not strongly opposed to the other way, it's easy to flip either way in the future based on more experience of using these commands

Reply:
consistency++

Reply:
`test-account-external` is added here. but i see ya! consistency with the existing identity makes the most sense. gonna flip what i have here so that `test-*` is implicitly external only. `test-internal-*` should run... all tests or _only_ internal? my preference is for all.

Reply:
not a huge opinion on that, could be all and if so changing identity internal to do the same

Reply:
updated w/ following comment:
```make
# Make commands for testing should have the following pattern:
# `test-{service}` should run tests that use only external-facing client functions
# `test-internal-{service}` should run all tests, including ones calling internal endpoints
# Tests that make use of internal endpoints must be prefixed "TestInternal".
```